### DISCUSSION TITLE
Windows客户端0.4.3版本，安装后在某些情况下会闪屏

### DISCUSSION THREAD

**Discussion by szk7903** [qier222/YesPlayMusic#1181] [Q&A]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 用的是网页版还是客户端
- 浏览器名称或电脑操作系统
- 控制台 Console 页面的截图（按 F12 可打开控制台）

**Comment by memorydream** [comment]
具体是什么情况？触发条件呢？我没有碰到过

**Comment by szk7903** [comment]
我将遇到的问题拍成视频打包发送给你，麻烦看一下

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: memorydream ***@***.***>
发送时间: Tuesday, January 4, 2022 5:14:47 PM
收件人: qier222/YesPlayMusic ***@***.***>
抄送: szk7903 ***@***.***>; Author ***@***.***>
主题: Re: [qier222/YesPlayMusic] Windows客户端0.4.3版本，安装后在某些情况下会闪屏 (Issue #1174)


具体是什么情况？我没有碰到过

―
Reply to this email directly, view it on GitHub<https://github.com/qier222/YesPlayMusic/issues/1174#issuecomment-1004640523>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQMVHUOTKCEJLADKRMIIUJDUUK3APANCNFSM5LGP57RQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by memorydream** [comment]
能直接发在github上吗，我这边没看到你发的视频

**Comment by szk7903** [comment]
[](url
[视频文件.zip](https://github.com/qier222/YesPlayMusic/files/7807175/default.zip)
)

**Comment by memorydream** [comment]
尝试禁用gpu加速试试？

**Comment by pan93412** [comment]
比較像是驅動程式或顯示卡的問題。
